custom/enikshay/case_utils.py

```diff
@@ -203,15 +201,24 @@ def get_open_drtb_hiv_case_from_episode(domain, episode_case_id):
 
 def get_open_episode_case_from_person(domain, person_case_id):
```

Comment:
it looks like this is used all over the code base. have you verified that this change is ok for all of them?